jbig2/jbig2_compress_image_checkboard.go

```diff
@@ -0,0 +1,91 @@
+/*
+ * This example showcases the conversion of the jpg encoded image
+ * into jbig2 encoding format.
+ *
+ * The result jbig2 file is compressed with a lossless method stored with
+ * the standard  jbig2 - .jb2 extension. The file compressed in this example
+ * is stored in just 377 bytes - compared to the 142 794 bytes for original image.
+ * This gives compression ratio (uncompressed size/compressed size) of 378.76, which leads to 99.735%
+ * space savings for given example.
+ */
+
+package main
+
+import (
+	"fmt"
+	"image"
+	"log"
+	"path/filepath"
+
+	// load jpeg decoder
+	_ "image/jpeg"
+	"os"
+
+	"github.com/unidoc/unipdf/v3/core"
+)
+
+func main() {
+	// Let's read an jpeg rgba image from the file, convert it into JBIG2Image
```

Comment:
Let's not assume gopath..  Is this really only specific to checkerboard or can we allow any image.  But say
> For example, checkerboard-squares..jpg which is in folder xxx/ .. gives certain results
Allow the user to specify input file but explain for specific file

Reply:
I agree. Done.